x-pack/platform/plugins/private/reporting/server/services/scheduled_reports/scheduled_reports_service.ts

```diff
@@ -165,6 +167,7 @@ export class ScheduledReportsService {
         type: SCHEDULED_REPORT_SAVED_OBJECT_TYPE,
         page,
         perPage: size,
+        search,
```

Comment:
Could you please update the unit tests accordingly? 👀

Reply:
+1